Generator/Templates/Core/class.sbntxt

```diff
@@ -0,0 +1,18 @@
+using System;
+using GObject;
+
+namespace {{ namespace }}
+{
+    // AUTOGENERATED FILE - DO NOT MODIFY
+    
+    [Wrapper("{{ type_name }}")]
+    public partial class {{ name }} : {{ parent }}
+    {
+        protected internal new static GObject.Type GetGType() => new GObject.Type(Sys.{{ name }}.get_type());
```

Comment:
The return type here (GObject.Type) mismatch with the one in the type system (GObject.Sys.Type)

Reply:
This one is intentional. We want to either move `Sys.Type` -> `GObject.Type` or return a `ulong` directly I believe. We'll need to do this before we're ready to merge

Reply:
Yes this is correct. I would say we do not expose either `ulong` or `Sys.Type` as a Type just `GObject.Type`. Otherwise it would probably be confusing.